velox/common/caching/SsdFile.h

```diff
@@ -479,7 +474,7 @@ class SsdFile {
 
   // Returns true if checkpoint is needed.
   bool needCheckpoint(bool force) const {
-    if (!checkpointEnabled()) {
+    if (!checkpointEnabled() || checkpointWriteFile_ == nullptr) {
```

Comment:
Let's keep it the same and delete checkpoint file whenever we write a new checkpoint file? Thanks!